.circleci/config.yml

```diff
@@ -112,7 +112,7 @@ jobs:
             gem update bundler
             bundle update --bundler
             export BUNDLE_GEMFILE=Gemfile.next
-            bundle install --jobs=4 --retry=3 --path vendor/bundle
+            bundle update --jobs=4 --retry=3
```

Comment:
I'm comfused about this change, we will not install the gems but update them everytime this job runs?

Reply:
Ah forgot that bundle update installs stuff... Do we still want this? Isn't this job updating gems everytime could lead to underterministic results for the CI?

Reply:
@bronzdoc This is core to the "stay up to date" service: https://www.fastruby.io/stay-up-to-date

Considering that we want to get the latest from Rails `main` and then run the test suite against that, I think this is the best way to do it. I don't think `bundle install` will go and fetch the latest changes with `bundle install` 

The unique value proposition of our service is that we will figure out what changes in Rails `main` will break your application code (_undeterministic results_) and fix them. 😸

Reply:
Make sense, thanks for the clarification @etagwerker